Makefile

```diff
@@ -142,8 +142,16 @@ test-all:
 install: clean
 	python3 setup.py install
 
+.PHONY: eject-contracts
+eject-contracts:
+	@for contract in component_registry registries_manager service_registry ; do \
+		echo Updating $$contract contract; \
+    	rm -rf autonomy/data/contracts/$$contract ; \
+		cp -r packages/valory/contracts/$$contract autonomy/data/contracts/$$contract ; \
+	done
```

Comment:
Include contract packages in the `data/` directory instead of installing them as a separate package